README.md

```diff
@@ -21,6 +21,12 @@ use {'romgrk/fzy-lua-native', run = 'make'}
 use {'tzachar/fuzzy.nvim', requires = {'romgrk/fzy-lua-native'}}
 ```
 
+Using [Packer](https://github.com/wbthomason/packer.nvim/) with `zf`:
+```lua
+use {'natecraddock/telescope-zf-native.nvim'}
```

Comment:
This is not needed.
If you don't need to add any special config, just adding the dependency to `requires` is enough.